Flickr/Sources/FlickrAppVCs/SearchResultsVC.swift

```diff
@@ -6,17 +6,200 @@
 //
 
 import Foundation
+import Nuke
 import UIKit
 
+struct Photos: Codable {
+    var photos: PhotoArray
+}
+
+struct PhotoArray: Codable {
+    var photo: [SinglePhoto]
+
+    func getPhotoArray() -> [SinglePhoto] {
+        photo
+    }
+}
+
+struct SinglePhoto: Codable {
+    var id: String
+    var owner: String
+    var secret: String
+    var server: String
+    var title: String
+}
+
 class SearchResultsVC: UIViewController {
+    private var photos: [URL] = []
+    private var imageTitles: [String] = []
+    private var searchString: String?
+    private var collectionView: UICollectionView?
+    private var constants = GlobalConstants()
+    weak var searchResultsDelegate: SearchResultsViewControllerDelegate?
+
+    init(searchString: String) {
+        self.searchString = searchString
+        super.init(nibName: nil, bundle: nil)
+    }
+
+    @available(*, unavailable)
+    required init?(coder _: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    func returnImageView(cell _: UICollectionViewCell, indexPath _: IndexPath) -> UIImageView {
+        let imageView = UIImageView()
+        imageView.configureView { imageView in
+            imageView.tag = 10
+            imageView.contentMode = .scaleAspectFill
+            imageView.clipsToBounds = true
+        }
+        return imageView
+    }
+
+    func returnImageViewConstraints(imageView: UIImageView, cell: UICollectionViewCell) -> [NSLayoutConstraint] {
+        [
+            imageView.leadingAnchor.constraint(equalTo: cell.leadingAnchor),
+            imageView.trailingAnchor.constraint(equalTo: cell.trailingAnchor),
+            imageView.topAnchor.constraint(equalTo: cell.topAnchor),
+            imageView.bottomAnchor.constraint(equalTo: cell.bottomAnchor)
+        ]
+    }
```

Comment:
Why is this setup inside the ViewController? Please create a custom CollectionViewCell to set this up?